src/mudlet.pro

```diff
@@ -761,6 +762,10 @@ LUA_GEYSER.files = \
     $${PWD}/mudlet-lua/lua/geyser/GeyserWindow.lua
 LUA_GEYSER.depends = mudlet
 
+LUA_TRANSLATIONS.files = \
+    $${PWD}/../translations/lua/*
```

Comment:
:-1: You *cannot* (the last time I checked) wild-card the file names when making a **Unix** GNU Makefile in QMake so that it can do a `make install` - you have to list each file individually (see all the 250+ files that I had to put in to support the Lua Code Formatter :nauseated_face:) ...

Reply:
@Githzerai could you confirm this?

Reply:
make install is working as expected in Ubuntu 19.10
Don't know about other system though.

Reply:
We don't need the 250 files listed explicitly in the `.pro` actually? @SlySven ?

Reply:
You used to have to - and I have not seen anything to suggest that has changed...